<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="replaceToLabel">
       <property name="text">
        <string>Replace by:</string>
       </property>
       <property name="buddy">
        <cstring>replaceToLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textToFindLabel">
       <property name="text">
        <string>Text to find:</string>
       </property>
       <property name="buddy">
        <cstring>textToFindLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="textToFindLineEdit">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Enter text to find</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="replaceToLineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter text to replace to</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="searchBackwardCheckBox">
          <property name="text">
           <string>Search &amp;backward</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="caseSensitiveCheckBox">
          <property name="text">
           <string>&amp;Case sensitive</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="findWholeWordsCheckBox">
          <property name="text">
           <string>&amp;Whole words</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="findNextButton">
       <property name="text">
        <string>&amp;Find Next</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAllButton">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Clos&amp;e</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionFindNext">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>FindNext</string>
   </property>
   <property name="toolTip">
    <string>Find/Find Next</string>
   </property>
   <property name="shortcut">
    <string>Enter</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionReplaceAll">
   <property name="text">
    <string>ReplaceAll</string>
   </property>
   <property name="toolTip">
    <string>Replace All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ReplaceDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findNextButton</sender>
   <signal>clicked()</signal>
   <receiver>actionFindNext</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceButton</sender>
   <signal>clicked()</signal>
   <receiver>actionReplace</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceAllButton</sender>
   <signal>clicked()</signal>
   <receiver>actionReplaceAll</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>find(QString)</signal>
 </slots>
</ui>
